base_mail_bcc/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+BCC all emails
+==============
+
+This module extends the email meachnism to allow for sending a blind carbon copy (BCC) 
+of all outgoing emails to configurable e-mail addresses.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to Settings > Parameters > System Parameters
+* Create a new entry with key `mail.always_bcc_to` and set the desired e-mail addresses for BCC as value. This value must be comma-separated list of valid e-mail addresses.
```

Comment:
`mail.always_bcc_to` - this implies the parameter has something to do with the mail module, which is not completely the case. I propose to always prefix keys with your module name, so in your case, I'd call the key `base_mail_bcc.bcc_to`